docs/shell.rst

```diff
@@ -0,0 +1,141 @@
+=====================================================
+InSpec Shell Usage
+=====================================================
+
+The InSpec interactive shell is a pry based REPL that can be used to quickly run InSpec controls and tests without having to write it to a file.  Its functionality is similar to ``chef shell`` - it provides a way to exercise the InSpec DSL, its resources, tests and plugins without having to create a profile or write a test file.
+
+Launching the shell
+-----------------------------------------------------
+
+If you are using InSpec from a platform-specific package (rpm, msi, etc.) or from a chef shell in ChefDK, you can directly launch chef shell against your local machine using:
+
+.. code-block:: bash
+
+    $ inspec shell
+    $ inspec help shell # This will describe inspec shell usage
+
+If you wish to connect to a remote machine (called a target within InSpec), you can use the ``-t`` flag.  We support connecting using ssh, WinRm and docker.  For an ssh connection, use ``-i`` for specifying ssh key files, and the ``--sudo*`` commands for requesting a privelege escalation after logging in.  For a WinRM connection, use ``--path`` to change the login path, ``--ssl`` to use SSL for transport layer encryption.
```

Comment:
I'm not entirely sure how to add references to other parts of the doc.  I've never use rst before.